app/layer_example/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(ARM_DIR "/3rdparty/ComputeLibrary")

```

Comment:
This is an absolute path. You need to use CMAKE_SOURCE_DIR or other variable to reach what you want